src/components/status-check.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Box, Flex, space, themeColor, color } from '@blockstack/ui';
+import { border } from '@common/utils';
+import { Link } from '@components/mdx';
+import { Text } from '@components/typography';
+import fetch from 'isomorphic-unfetch';
+import { CheckmarkCircleIcon, ExclamationMarkCircleIcon } from '@blockstack/ui';
+
+export const StatusCheck: React.FC<any> = () => {
+  const [status, setStatus] = React.useState(false);
+  const STATUS_CHECKER_URL = 'https://status-checker-blockstack.herokuapp.com';
+
+  React.useEffect(() => {
+    fetch(`${STATUS_CHECKER_URL}/json`)
+      .then(r => r.json())
+      .then(data => {
+        if (data && data.masterNodePings.length > 1) {
+          setStatus(data.masterNodePings[0].value === 1);
+        }
+      })
+      .catch(err => console.log(err));
```

Comment:
Can we assume if the status check isn't working it's down? and `setStatus` here?

Reply:
I think it's possible the status checker could be down while the testnet is available lol

Reply:
@kyranjamie that makes sense to me